testsuite/classlibrary/TestScore.sc

```diff
@@ -0,0 +1,43 @@
+TestScore : UnitTest {
+
+	test_pattern_asScore_lag {
+		var events, score, onsetTimes;
+		events = [(x: 0, lag: 0.5), (x: 1, dur: 0.5), (x: 2, lag: 0.5)];
+		score = Pseq(events).asScore(3);
+		onsetTimes = score.score.drop(1).postcs.select { |x| x[2] == \default }.collect { |x| x.first };
```

Comment:
can you please remove the `postcs` from these tests?

Reply:
also, using `collect(_.first)` at the end might make it more readable. personally i think it would be clearest to break this into two lines with an extra variable

Reply:
ok, done!